ChameleonForms.AcceptanceTests/Helpers/HtmlHelpers.cs

```diff
@@ -3,17 +3,18 @@
 using System.Threading;
 using System.Threading.Tasks;
 using AngleSharp;
-using AngleSharp.Dom.Html;
-using AngleSharp.Network;
+using AngleSharp.Html.Dom;
+using AngleSharp.Io;
+using AngleSharp.Io.Network;
 
-namespace RazorPagesProject.Tests.Helpers
+namespace ChameleonForms.AcceptanceTests.Helpers
 {
     public class HtmlHelpers
     {
-        public static async Task<IHtmlDocument> GetDocumentAsync(HttpResponseMessage response)
+        public static async Task<IHtmlDocument> GetDocumentAsync(HttpClient client, HttpResponseMessage response)
         {
             var content = await response.Content.ReadAsStringAsync();
-            var document = await BrowsingContext.New()
+            var document = await BrowsingContext.New(Configuration.Default.WithRequester(new HttpClientRequester(client)).WithDefaultLoader())

```

Comment:
This is needed so navigation calls within Anglesharp use the HttpClient that talks to the in-memory test server rather than issuing real requests to http://localhost/ :P